ui/src/components/yaml/index.tsx

```diff
@@ -45,6 +45,7 @@ const Yaml = (props: IProps) => {
   const yamlRef = useRef<LegacyRef<HTMLDivElement> | undefined>()
```

Comment:
This line adds an unnecessary ref `contentRef` which is not used anywhere in the code. It should be removed to avoid dead code.